cfrontend/C2C.ml

```diff
@@ -76,13 +76,13 @@ let atom_sections a =
   with Not_found ->
     []
 
-let atom_is_small_data a ofs =
```

Comment:
The reason for passing the offset to "atom_is_small_data" is that, conceivably, "global + small offset" could be addressable via the SDA while "global + very large offset" might not, requiring normal addressing.  So, the axiomatization of SDA addressing in powerpc/Asm.v takes this possibility into account.  In reality, the compiler has no good way to know whether the offset causes SDA addressing to fail or not, which is why the "ofs" argument is actually ignored.